command_line/show.py

```diff
@@ -251,15 +251,14 @@ def show_experiments(experiments, show_scan_varying=False):
     text = []
     for i_expt, expt in enumerate(experiments):
         text.append("Experiment %i:" % i_expt)
-        format_class = expt.imageset.get_format_class()
-        if not format_class.is_abstract():
-            text.append(f"Format class: {format_class.__name__}")
+        template = None
+        if expt.imageset:
+            format_class = expt.imageset.get_format_class()
+            template = expt.imageset.get_template()
```

Comment:
Is `get_template()` defined for _all_ `ImageSet`s, or is it just an `ImageSequence` thing? If there wasn't explicit support already for `expt.imageset` being `None` then I suspect the `try`/`except AttributeError` was there for a reason...

Reply:
History repeats itself 😄 
https://github.com/dials/dials/pull/1133#discussion_r374706496

Reply:
well spotted :-) I'll put that back then